core/input/input_enums.h

```diff
@@ -56,7 +56,7 @@ enum class JoyAxis {
 	TRIGGER_LEFT = 4,
 	TRIGGER_RIGHT = 5,
 	SDL_MAX = 6,
-	MAX = 10, // OpenVR supports up to 5 Joysticks making a total of 10 axes.
+	MAX = 10, // OpenVR supports up to 5 Joysticks making a total of 10 axes. Changing this value affects JoyButton::MAX.
```

Comment:
How does it affect JoyButton?

Reply:
I made it more clear in the updated comment.